modules/telco-ran-red-hat-advanced-cluster-management-rhacm.adoc

```diff
@@ -10,10 +10,10 @@ New in this release::
 * Additional node labels can be configured during installation.
 
 Description::
-{rh-rhacm} provides Multi Cluster Engine (MCE) installation and ongoing lifecycle management functionality for deployed clusters.
+{rh-rhacm} provides the {mce} installation and ongoing lifecycle management functionality for deployed clusters.
```

Comment:
Are you sure that "operator installation" will be understood correctly here as installing the MCE itself, nit just making its operator available for a separate installation?

Reply:
I think it's fine because the cluster lifecycle function is provided by the multicluster engine for Kubernetes Operator, which is installed automatically with RHACM